direct.go

```diff
@@ -108,79 +113,138 @@ func (d *direct) loadCandidates(initial map[string][]*Masquerade) {
 	}
 }
 
-func (d *direct) vetInitial(numberToVet int) {
+// Vet vets the specified Masquerade, verifying certificate using the given CertPool
+func Vet(m *Masquerade, pool *x509.CertPool) bool {
```

Comment:
Oh, also @oxtoacart does some external package call this? Just curious about it being public.